### PR TITLE
Updated nav bar

### PR DESCRIPTION
**Pull request by beckystar4**
## Trello Card

https://trello.com/c/Nvr81EFW/53-change-nav-bar-to-change-when-you-are-logged-in-vs-not
https://trello.com/c/FYcXMACV/54-when-logged-in-should-be-able-to-access-ingredient-inventory-and-recipe-library

## Description

Updated the nav bar so if the user is logged in they can see the liked recipe library/ingredient inventory

### CODE REVIEW

File: app/views/layouts/_navbar.html.erb [hunk 1]
```diff
@@ -27,10 +26,22 @@
                         <%= link_to('Recipes', recipes_path, class: "nav-link#{' active' if controller_name == 'recipes'}", aria_current: controller_name == 'recipes' ? 'page' : false) %>
                     </li>
 
-                    <%# TODO: Add link %>
+                    <% if logged_in? %>
```
Comment (JMGbaum) [inline app/views/layouts/_navbar.html.erb:29]: Where is this `logged_in?` method defined?

File: app/views/layouts/_navbar.html.erb [hunk 2]
```diff
@@ -27,10 +26,22 @@
...
+                    </li>
+                    <li class="nav-item">
+                        <%= link_to('Inventory', inventory_items_path, class: "nav-link#{' active' if controller_name == 'ingredient_items'}", aria_current: controller_name == 'ingredient_items' ? 'page' : false) %>
+                    </li>
                     <li class="nav-item">
                         <%= link_to('My Profile', '#', class: "nav-link#{' active' if controller_name == 'users'}", aria_current: controller_name == 'users' ? 'page' : false) %>
                     </li>
+                    
+                    <% else %>
+                     <%= link_to('Login', login_path, class: "nav-link#{' active' if controller_name == 'users'}", aria_current: controller_name == 'users' ? 'page' : false) %>
```
Comment (JMGbaum) [inline app/views/layouts/_navbar.html.erb]:
```suggestion
                     <%= link_to('Login', login_path, class: "nav-link#{' active' if current_page?(login_path) || current_page?(signup_path)}", aria_current: current_page?(login_path) || current_page?(signup_path) ? 'page' : false) %>
```
We want to make sure we're only adding the `active` class on the associated pages.

File: app/views/layouts/_navbar.html.erb [hunk 3]
```diff
@@ -27,10 +26,22 @@
...
-                    <%# TODO: Add link %>
+                    <% if logged_in? %>
+                    <li class="nav-item">
+                        <%= link_to('Liked Recipes', librarys_path, class: "nav-link#{' active' if controller_name == 'librarys'}", aria_current: controller_name == 'librarys' ? 'page' : false) %>
+                    </li>
+                    <li class="nav-item">
+                        <%= link_to('Inventory', inventory_items_path, class: "nav-link#{' active' if controller_name == 'ingredient_items'}", aria_current: controller_name == 'ingredient_items' ? 'page' : false) %>
+                    </li>
                     <li class="nav-item">
                         <%= link_to('My Profile', '#', class: "nav-link#{' active' if controller_name == 'users'}", aria_current: controller_name == 'users' ? 'page' : false) %>
```
Comment (JMGbaum) [inline app/views/layouts/_navbar.html.erb:37]: We'll need to change the "my profile" link once we create an appropriate controller action.

File: app/views/layouts/_navbar.html.erb (hunk 1 above)
Comment (beckystar4) [reply app/views/layouts/_navbar.html.erb:29]: sessions_helper.rb

**Pull request merged**